pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "dbt-dry-run"
-version = "0.6.2"
+version = "0.6.3"
```

Comment:
I like to do a separate [release commit](https://github.com/autotraderuk/dbt-dry-run/commit/804c7fbe7a84e350bd17cad7f333c9decb4d80a9) to bump the version.

I can show you how to trigger a release when this PR is merged though.

Reply:
Thanks